modules/mobile/docs/openapi.yaml

```diff
@@ -269,6 +389,152 @@ paths:
       security:
         - Bearer: []
       summary: /v0/appointments/cancel/{cancelId}
+  /v0/appointments/check-in:
```

Comment:
I initially had appointment IEN as part of path but IEN, while it is a unique identifier for the appt, is different from the appt ID that we already and using anything except the appointment id in path seemed incorrect.

Reply:
Yeah I think keeping that in the body is good